### PR TITLE
Added Project (BMI Calculator)

### LINKED ISSUE #1475: chore: project addition by aparna03dubey
### Add a new project to the list

This project is a Body Mass Index, made using HTML, CSS and Javascript.

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I'm a GSSoC'23 contributor
- [X] I want to work on this issue
- [X] My project has contribution guidelines
- [X] My project has a Code of conduct
- [X] My project has README
- [X] My project has a License

### PR DESCRIPTION
**Pull request by aparna03dubey**
## Related Issue

Fixes #1475 
## Description

My Project calculates Body mass index of an Indivisual by taking height and weight as input.

## Screenshots
BMI

## Checklist for projects addition

 

- [Y ] My project has contribution guidelines.
- [ Y] My project has a Code of Conduct.
- [ Y] My project has a README file.
- [ Y] My project has a License.

---

Please note that if your project doesn't meet these requirements, your PR may be closed. Make sure to include these elements in your project before submitting the PR.

### CODE REVIEW

**Comment by aparna03dubey** [comment]
I have added my project.

**Comment by priyankarpal** [comment]
this branch has a conflicts error. Please resolve it

File: src/DB/projects.json [hunk 1]
```diff
@@ -2338,5 +2338,35 @@
...
+  },
+    {
+    "github_username": "aparna03dubey",
+    "Social_media": {
+      "gitHub": "https://github.com/aparna03dubey",
+      "LinkedIn": "https://www.linkedin.com/in/aparna-dubey-941887225/"
+    },
+    "Projects": [
+      {
+        "link": "https://aparna03dubey.github.io/Water-Tracker.github.io/",
```
Comment (priyankarpal) [inline src/DB/projects.json:2350]: add github repo link

File: src/DB/projects.json [hunk 2]
```diff
@@ -2338,5 +2338,35 @@
...
+  }
+    {
+    "github_username": "aparna03dubey",
+    "Social_media": {
+      "gitHub": "https://github.com/aparna03dubey",
+      "LinkedIn": "https://www.linkedin.com/in/aparna-dubey-941887225/"
+    },
+    "Projects": [
+      {
+        "link": "https://aparna03dubey.github.io/MyBMICalculator/",
```
Comment (priyankarpal) [inline src/DB/projects.json:2365]: add github repo link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**